docker-compose.override.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+
+services:
+    hub-node:
+        volumes:
+            - ./moj-node/server:/home/node/app/server
```

Comment:
Do these volumes need mounting?  I can't find any volumes mounted in either staging or berwyn at the moment.

Reply:
As discussed these mounts are for local testing before staging.